scripts/integration-test.sh

```diff
@@ -1,6 +1,531 @@
+  echo "      * the processor used Processor2015 with the MozillaProcessorAlgorithm2015"
+  socorro collector2015 \
+      --services.services_controller='[{"name": "collector", "uri": "/submit", "service_implementation_class": "socorro.collector.wsgi_breakpad_collector.BreakpadCollector2015"}, {"name": "pellet_stove", "uri": "/pellet/submit", "service_implementation_class": "socorro.collector.wsgi_generic_collector.GenericCollector"}]' \
+      --services.collector.storage.crashstorage_class=socorro.external.crashstorage_base.PolyCrashStorage \
+      --services.collector.storage.storage_classes='socorro.external.fs.crashstorage.FSPermanentStorage, socorro.external.rabbitmq.crashstorage.RabbitMQCrashStorage' \
+      --services.collector.storage.storage0.crashstorage_class=socorro.external.fs.crashstorage.FSPermanentStorage \
+      --services.collector.storage.storage1.crashstorage_class=socorro.external.rabbitmq.crashstorage.RabbitMQCrashStorage \
+      --services.pellet_stove.storage.crashstorage_class=socorro.external.fs.crashstorage.FSPermanentStorage \
+      --services.pellet_stove.storage.fs_root=./submissions \
+      --resource.rabbitmq.host=$rmq_host \
+      --secrets.rabbitmq.rabbitmq_user=$rmq_user \
+      --secrets.rabbitmq.rabbitmq_password=$rmq_password \
+      --resource.rabbitmq.virtual_host=$rmq_virtual_host \
+      --resource.rabbitmq.standard_queue_name=$rmq_normal_queue_name \
+      --resource.rabbitmq.priority_queue_name=$rmq_priority_queue_name \
+      --resource.rabbitmq.reprocessing_queue_name=$rmq_reprocessing_queue_name \
+      --resource.rabbitmq.routing_key=$rmq_normal_queue_name \
+      --resource.rabbitmq.transaction_executor_class=socorro.database.transaction_executor.TransactionExecutor \
+      --web_server.wsgi_server_class=socorro.webapi.servers.CherryPy \
+      --web_server.ip_address=0.0.0.0 \
+      > collector.log 2>&1 &
+  echo '   collector started'
+  socorro processor \
+      --resource.rabbitmq.host=$rmq_host \
+      --secrets.rabbitmq.rabbitmq_user=$rmq_user \
+      --secrets.rabbitmq.rabbitmq_password=$rmq_password \
+      --resource.rabbitmq.virtual_host=$rmq_virtual_host \
+      --resource.rabbitmq.standard_queue_name=$rmq_normal_queue_name \
+      --resource.rabbitmq.priority_queue_name=$rmq_priority_queue_name \
+      --resource.rabbitmq.reprocessing_queue_name=$rmq_reprocessing_queue_name \
+      --resource.rabbitmq.routing_key=$rmq_normal_queue_name \
+      --resource.postgresql.database_hostname=$database_hostname \
+      --secrets.postgresql.database_username=$database_username \
+      --secrets.postgresql.database_password=$database_password \
+      --new_crash_source.new_crash_source_class='socorro.external.rabbitmq.rmq_new_crash_source.RMQNewCrashSource' \
+      --processor.processor_class='socorro.processor.mozilla_processor_2015.MozillaProcessorAlgorithm2015' \
```

Comment:
the original used quotes that were unnecessary, I was inconsistent in cleaning them up.